app/helpers/application_helper.rb

```diff
@@ -597,4 +597,11 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def speaker_selector_input(form)
+    users = User.select(:id, :name, :email).where(is_disabled: false).order(:name)
+    form.input :speakers, as: :select,
+                          collection: options_for_select(users.map {|user| ["#{user.name} (#{user.email})", user.id]}, @event.speakers.map(&:id)),
```

Comment:
A lot of people prefer to refer to other by their username, so i will add that as well